docs/changelog-fragments.d/957.internal.md

```diff
@@ -0,0 +1,9 @@
+Added new subcommand *settings* to output all current ansible-navigator
```

Comment:
Should we use an inline code highlighting instead of _italics_ here?
```suggestion
Added a new subcommand `:settings` to output all current ansible-navigator
```

Reply:
Yes we should. I updated this.